### PR TITLE
feat: add dailymotion support

### PR DESCRIPTION
**Pull request by dumbmoron**
i gotta be a deleted youtube video the way im always on some dailymotion

### CODE REVIEW

**Comment by Rashedaldahmani** [comment]
What does this mean?

**Comment by wukko** [comment]
> What does this mean?

what exactly?

**Comment by Rashedaldahmani** [comment]
How will this effect cobalt?

File: src/test/tests.json [hunk 1]
```diff
@@ -1180,5 +1180,30 @@
...
+        "name": "regular video",
+        "url": "https://www.dailymotion.com/video/x8t1eho",
+        "params": {},
+        "expected": {
+            "code": 200,
+            "status": "stream"
+        }
+    }, {
+        "name": "private video",
+        "url": "https://www.dailymotion.com/video/k41fZWpx2TaAORA2nok",
```
Comment (wukko) [inline src/test/tests.json:1194]: please use another link from an account with an appropriate name :(

Comment (Rashedaldahmani) [reply src/test/tests.json:1194]: Is it available now?

Comment (wukko) [reply src/test/tests.json:1194]: not yet

**Review by wukko — APPROVED** [review]

**Comment by dumbmoron** [comment]
merged as a97733d257d5ee2dd54533eb21422b3f8fa0d215

**Pull request closed without merging**